.github/workflows/wordlist.txt

```diff
@@ -0,0 +1,18 @@
+AJR
+CrCl
+EF
+HC
+HN
+ICI
+JZ
+LW
+OCTN
+PMAT
+Ren
+situ
+SLC
+tid
+acebutolol’s
+atenolol’s
+pindolol’s
+sotalol’s

```

Comment:
some abbreviations etc.
spell check will stil lfail due to some errors in the unused files (s. https://github.com/Open-Systems-Pharmacology/Qualification-CKD/issues/4), README.md etc.